README.md

```diff
@@ -1,18 +1,25 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
 # Global Conference for African Development
+>The summit's objectives are to address national adoption strategies and difficulties, choose the optimal operating model, and establish governance. Using the appropriate technologies for it is another.
 
 > This capstone project is about the demonstrating website development ability acquired from Microverse 4 weeks training.
 
 ## Built With
 
-- HTML, CSS, Javasript
+- HTML, CSS, JavaScript
 - Linters
 
+## Designed by 
+
+-  Cidny Shin is the one the credit goes to for creating such an elegant and simple design. You can refer the mockup [here](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)
 ## Live Demo (if available)
 
 [Live Demo Link](https://akos29.github.io/conference/)
 
+## Loom Video
+
+- Please refer to my brief [demonstration](https://www.loom.com/share/fe99979688214cc8b3bd83a11b8214f9) about the website. Thank you in advance for watching.
```

Comment:
- [ ] Kindly highlight some interesting piece of code or something you built that you are very proud of.